interface/wx/bmpbndl.h

```diff
@@ -293,6 +296,19 @@ class wxBitmapBundle
     /// @overload
     static wxBitmapBundle FromSVG(const wxByte* data, size_t len, const wxSize& sizeDef);
 
+    /**
```

Comment:
Is making the two rasterizer-specific functions public the best idea?

For example, we do not allow the NanoSVG rasterizer to be created with `NSVGimage`...